src/openrct2/localisation/string_ids.h

```diff
@@ -3647,6 +3647,16 @@ enum {
 	STR_BOOSTER = 5995,
 	STR_RIDE_CONSTRUCTION_BOOSTER_SPEED = 5996,
 
```

Comment:
Marijn lost them during refactoring and didn't feel like putting them back. But they should be there, in order to keep related strings together.